### ISSUE TITLE
Urql fails with "no exported member 'QueryProps'.  TS2694"

### ISSUE THREAD

**Issue by pnordhus** [dotansimha/graphql-code-generator#1995]
**Describe the bug**
 
When using the typescript-urql plugin, any autogenerated file fails compiling with:
```
TypeScript error in /home/user/testapp/src/graphql.tsx(30,20):
Namespace '"/home/user/testapp/node_modules/urql/dist/types/index"' has no exported member 'QueryProps'.  TS2694

    28 | 
    29 | export const TestComponent = (
  > 30 |   props: Omit<Urql.QueryProps, "query"> & { variables?: TestQueryVariables }
       |                    ^
    31 | ) => <Urql.Query {...props} query={TestDocument} />;
    32 | 
```

**To Reproduce**

1. My GraphQL schema:

```graphql
type Query {
        test: String
}
```

2. My GraphQL operations:

```graphql
query test {
        test
}
```

3. My `codegen.yml` config file:

```yml
schema: schema.graphql
generates:
  src/graphql.tsx:
    documents:
      - test.graphql
    plugins:
      - typescript
      - typescript-operations
      - typescript-urql
```

**Expected behavior**

The autogenerated file `src/graphql.tsx` should compile.

**Environment:**

- OS: Arch Linux
- `@graphql-codegen/...`: 1.2.1
- NodeJS: 10.16.0
- Typescript: 3.5.1
- Urql: 1.1.1

**Comment by pnordhus** [comment]
This issue only occurs when generating React Components. When I disable Component generation with
```yaml
withComponent: false
```
the generated Typescript file can be compiled. Generated Hooks work as well.

**Comment by dotansimha** [comment]
Hi @pnordhus ! Thank you for reporting that, and thanks for the workaround. 
It seems like an issue with `urql`, I opened an issue for that: https://github.com/FormidableLabs/urql/issues/309

**Comment by dotansimha** [comment]
@pnordhus The awesome @FormidableLabs fixed that very quickly and released `urql@1.1.2` with a fix for that.

**Comment by pnordhus** [comment]
Thanks @dotansimha and @FormidableLabs for the quick fix! This issue is indeed fixed, however `withComponent: true` does still not work, with a different error. I'll create a new issue.
